src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -362,8 +468,10 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
 }
 
 // italicize the delivery instructions in the base address component
-::v-deep .address-block__info-row:last-of-type {
-  padding-top: 0.75rem;
-  font-style: italic;
+::v-deep {
+  .address-block {
```

Comment:
If you're targeting only the delivery instructions then you'll need to use `::v-deep .base-address .address-block .delivery-instructions`... otherwise, the comment on line 470 is wrong.

Reply:
For comparison: https://github.com/bcgov/business-create-ui/blob/299527f5e6b0f353f0b16f0891eb2ce52d547f52/src/components/common/ListPeopleAndRoles.vue#L316

Reply:
Comment is obsolete, carry over. The component itself now provides the styling that was being implemented here(previously)
Removed comment

Reply:
Please add comment describing what this override is doing.

Reply:
Updated